gpytorch/kernels/distributional_input_kernel.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+from torch.distributions import Normal, kl_divergence
+
+from .kernel import Kernel
+
+
+def _symmetrized_kl(dist1, dist2, num_dims=2, eps=1e-8):
+    dist1_mean = dist1[..., :num_dims].unsqueeze(-3)
```

Comment:
Would we ever have the case where `num_dims != dist1.shape[-1] // 2`? If not, we can get rid of the arg and just infer it this way to simplify the interface.

Reply:
Not entirely sure. For other distributions, very possibly?